tests/test_metric_cards.py

```
@@ -24,15 +24,14 @@ def compose(self) -> ComposeResult:
 
 @pytest.mark.asyncio
 async def test_metric_card_displays_value():
-    """MetricCard should display label and value."""
+    """MetricCard should display only the stat value in card content."""
     app = MetricCardTestApp()
     async with app.run_test() as pilot:
```

Ruff (0.15.2)

[error] Local variable `pilot` is assigned to but never used

Remove assignment to unused variable `pilot`

(F841)